src/pipelines/vaccinations/us_authority_vaccination_type.py

```diff
@@ -0,0 +1,132 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+from functools import partial
+from lib.concurrent import process_map
+from typing import Dict
+from pandas import DataFrame, concat
+from lib.cached_data_source import CachedDataSource
+from lib.utils import table_rename
+from lib.io import read_file
+
+
+_column_adapter = {
+    'Series_Complete_Yes': 'total_persons_fully_vaccinated',
+    'Doses_Administered': 'total_vaccine_doses_administered',
+    'Administered_Dose1_Recip':'total_persons_vaccinated',
+
+    'Series_Complete_Pfizer': 'total_persons_fully_vaccinated_Pfizer',
+    'Administered_Pfizer': 'total_vaccine_doses_administered_Pfizer',
+
+    'Series_Complete_Moderna': 'total_persons_fully_vaccinated_Moderna',
+    'Administered_Moderna': 'total_vaccine_doses_administered_Moderna',
+
+    'Series_Complete_Janssen': 'total_persons_fully_vaccinated_Janssen',
+    'Administered_Janssen': 'total_vaccine_doses_administered_Janssen',
+
+    "key": "key"
+    }
+
+states = [
+ 'US',
+ 'AK',
+ 'AL',
+ 'AR',
+ 'AS',
+ 'AZ',
+ 'CA',
+ 'CO',
+ 'CT',
+ 'DC',
+ 'DE',
+ 'FL',
+ 'GA',
+ 'GU',
+ 'HI',
+ 'IA',
+ 'ID',
+ 'IL',
+ 'IN',
+ 'KS',
+ 'KY',
+ 'LA',
+ 'MA',
+ 'MD',
+ 'ME',
+ 'MI',
+ 'MN',
+ 'MO',
+ 'MP',
+ 'MS',
+ 'MT',
+ 'NC',
+ 'ND',
+ 'NE',
+ 'NH',
+ 'NJ',
+ 'NM',
+ 'NV',
+ 'NY',
+ 'OH',
+ 'OK',
+ 'OR',
+ 'PA',
+ 'PR',
+ 'RI',
+ 'SC',
+ 'SD',
+ 'TN',
+ 'TX',
+ 'UT',
+ 'VA',
+ 'VI',
+ 'VT',
+ 'WA',
+ 'WI',
+ 'WV',
+ 'WY']
```

Comment:
NIT: indent with 4 spaces and use black's style which should be
```
[
    'US',
    'AZ',
    ...
    'WY',
]
```

Alternatively, if you think this can be reused elsewhere, I would create a file named `src/data/us_cdc_locations.csv` with these values and read it here instead.